### PR TITLE
Allow for dishes to be created with one or more menus

### PR DESCRIPTION
**Pull request by AmberWilkie**
PT Story: https://www.pivotaltracker.com/story/show/131259147

Changes proposed in this pull request:
- Create association between dishes and menus
- Add checkbox to dish page that shows available menus
- Display menus associated with a dish on dish view
- Prettified checkbox
- Added launchy gem
- Tests for (all?) of this
- Added a number of step definitions, including show me the page

What I have learned working on this feature: 
- How to make has_and_belongs_to_many associations & migrations
- How to work with collection_check_boxes (including the need to sanitize parameters as an `[]`

Screenshots: 
screen shot 2016-09-30 at 5 50 43 pm

Ready for review!

### CODE REVIEW

File: features/step_definitions/dish_steps.rb [hunk 1]
```diff
@@ -1,4 +1,13 @@
 Then(/^I should be on the dish page for "([^"]*)"$/) do |name|
   dish = Dish.find_by(dish_name: name)
-  visit dish_path(dish)
+  expect(current_path).to eq dish_path(dish)
+end
+
+When(/^select "([^"]*)" from "([^"]*)"$/) do |menu_name, field_name|
+  page.check(menu_name)
```
Comment (tochman) [inline features/step_definitions/dish_steps.rb]: No need to pass in two attributes here, right?

File: app/models/dish.rb [hunk 2]
```diff
@@ -2,4 +2,6 @@ class Dish < ApplicationRecord
   validates_presence_of :dish_name
   validates_presence_of :dish_desc
   validates_presence_of :dish_price
+
+  has_and_belongs_to_many :menus
```
Comment (tochman) [inline app/models/dish.rb:6]: I can't find a unit spec for this.  Am I just blind?

**Review by tochman — CHANGES_REQUESTED** [review]

File: app/models/dish.rb (hunk 2 above)
Comment (AmberWilkie) [reply app/models/dish.rb:6]: I put in an extra line break for you :) It's in DB table in `dish_spec`

File: features/step_definitions/dish_steps.rb (hunk 1 above)
Comment (AmberWilkie) [reply features/step_definitions/dish_steps.rb]: refactored that step def

**Review by tochman — COMMENTED** [review]
Looks good. Great job.

**Pull request merged**